qtaf-core/src/main/resources/qtaf.json

```diff
@@ -46,17 +46,17 @@
       "scenarioImageEvidence": true,
       "customStatus": {
         "test": {
-          "passed": null,
-          "failed": null,
-          "pending": null,
-          "skipped": null
+          "passed": "PASSED",
+          "failed": "FAILED",
+          "pending": "PENDING",
+          "skipped": "SKIPPED"
         },
         "step": {
-          "passed": null,
-          "failed": null,
-          "pending": null,
-          "skipped": null,
-          "undefined": null
+          "passed": "PASSED",
+          "failed": "FAILED",
+          "pending": "PENDING",
+          "skipped": "SKIPPED",
+          "undefined": "UNDEFINED"
```

Comment:
n my opinion, `null` is the better default since the configuration map explicitly checks for `null` values and prints corresponding info/error messages.

Reply:
Set values to null